Sources/NIOHTTP2/HTTP2Error.swift

```diff
@@ -200,6 +200,16 @@ public enum NIOHTTP2Errors {
             self.streamID = streamID
         }
     }
+
+    /// An attempt was made to send trailers without setting END_STREAM on them.
+    public struct TrailersWithoutEndStream: NIOHTTP2Error {
```

Comment:
Not sure what the Swift convention is but should we have Error as suffix ?

Reply:
Given that this is namespaced in an enum and can only be found by typing `NIOHTTP2Errors.TrailersWithoutEndStream`, I don't think we need to add the word `Error` as well.